fineract-core/src/main/java/org/apache/fineract/infrastructure/core/data/ApiGlobalErrorResponse.java

```diff
@@ -58,219 +68,132 @@ public class ApiGlobalErrorResponse {
 
     private List<ApiParameterError> errors = new ArrayList<>();
 
-    public static ApiGlobalErrorResponse unAuthenticated() {
```

Comment:
Please do not reorder the methods, because this way it is nearly impossible to see what was changed!

Reply:
There was no reorder, except the constructor and the common initialiser moved to the first place.